test/exprtest/main.cpp

```diff
@@ -0,0 +1,12 @@
+#include <cstdlib>
+
+#include <gmock/gmock.h>
+#include <gtest/gtest.h>
+
+int main(int argc, char *argv[]) {
+  int seed = time(NULL) / 30;
```

Comment:
I suppose we can use --gtest_random_seed=... option, which defaults to 0 https://github.com/google/googletest/blob/master/googletest/src/gtest.cc#L4631